sdk/ml/azure-ai-ml/azure/ai/ml/operations/_model_operations.py

```diff
@@ -75,8 +84,9 @@ def __init__(
         self,
         operation_scope: OperationScope,
         operation_config: OperationConfig,
-        service_client: Union[ServiceClient052022, ServiceClient102021Dataplane],
+        service_client: Union[ServiceClient042023Preview, ServiceClient102021Dataplane],
```

Comment:
Do we need to update `_ml_client` file for modelOperations to use Apr API instead of May API?

Reply:
correct, in fact we should not add ServiceClient042023Preview , as we don't want to expose it yet